protocols/virtualfund/virtualfund_single_hop_test.go

```diff
@@ -205,30 +220,186 @@ func TestCrankAsAlice(t *testing.T) {
 
 	ok(t, err)
 	equals(t, waitingFor, WaitingForCompletePrefund)
-	assertStateSentTo(t, got, expectedSignedState, bob)
-	assertStateSentTo(t, got, expectedSignedState, p1)
+	assertStateSentTo(t, effects, expectedSignedState, bob)
+	assertStateSentTo(t, effects, expectedSignedState, p1)
 
 	// Manually progress the extended state by collecting prefund signatures
 	collectPeerSignaturesOnSetupState(o.V, my.role, true)
 
 	// Cranking should move us to the next waiting point, update the ledger channel, and alter the extended state to reflect that
 	// TODO: Check that ledger channel is updated as expected
-	oObj, got, waitingFor, err = o.Crank(&my.privateKey)
+	oObj, effects, waitingFor, err = o.Crank(&my.privateKey)
 	o = oObj.(*Objective)
 
 	p := consensus_channel.NewAddProposal(o.ToMyRight.Channel.Id, 2, o.ToMyRight.getExpectedGuarantee(), big.NewInt(6))
 	sp := consensus_channel.SignedProposal{Proposal: p}
-	assertProposalSent(t, got, sp, p1)
 	ok(t, err)
+	assertProposalSent(t, effects, sp, p1)
 	equals(t, waitingFor, WaitingForCompleteFunding)
 
 	// Check idempotency
 	emptySideEffects := protocols.SideEffects{}
-	oObj, got, waitingFor, err = o.Crank(&my.privateKey)
+	oObj, effects, waitingFor, err = o.Crank(&my.privateKey)
 	o = oObj.(*Objective)
 	ok(t, err)
-	equals(t, got, emptySideEffects)
+	equals(t, effects, emptySideEffects)
 	equals(t, waitingFor, WaitingForCompleteFunding)
+
+	// If Alice had received a signed counterproposal, she should proceed to postFundSetup
+	guaranteeFundingV := consensus_channel.NewGuarantee(big.NewInt(10), o.V.Id, alice.destination, p1.destination)
+	o.ToMyRight.Channel = prepareConsensusChannel(my.role, alice, p1, guaranteeFundingV)
+
+	oObj, effects, waitingFor, err = o.Crank(&my.privateKey)
+	o = oObj.(*Objective)
+
+	postFS := state.NewSignedState(o.V.PostFundState())
+	mySig, _ = postFS.State().Sign(my.privateKey)
+	_ = postFS.AddSignature(mySig)
+
+	ok(t, err)
+	equals(t, waitingFor, WaitingForCompletePostFund)
+	assertStateSentTo(t, effects, postFS, bob)
+}
+
+// TestCrankAsBob tests the behaviour from a end-user's point of view when they are a follower in the ledger channel
```

Comment:
This comment (along with the similar one for `TestCrankAsAlice`) seems to tie the vfo role `bob` to the `consensuschannel` role `follower`, but these are unrelated ideas.

Reply:
Yes, good point. I am fixing these in a follow-on: https://github.com/statechannels/go-nitro/pull/530 (just doing a self-review now)